wesnoth-units.po

```diff
@@ -6150,7 +6208,6 @@ msgstr ""
 
 #. [unit_type]: id=Orcish Nightblade, race=orc
 #: data/core/units/orcs/Nightblade.cfg:4
-#, fuzzy
```

Comment:
vážně vám ten (můj) syntetický výtvor připadá použitelný?
nebyl jsem si tím jistý

Reply:
Šeročepel je náhodou fajn. Temnočepel by znělo ještě víc synteticky. A něco jiného mě nenapadá. Noční podřezávač? Ale to také není nějak zvláštní.